toolkit/imageconfigs/linuxguard-amd64.yaml

```diff
@@ -119,72 +119,56 @@ os:
       - rd.luks=0
       - rd.hostonly=0
       - ipe.enforce=0
-      - systemd.journald.forward_to_console=1
 
   packages:
     remove:
-      - dracut-hostonly
-      - grub2-efi-binary
-      - kernel
-
+      - dracut-hostonly # Not used for UKI images
+      - grub2-efi-binary # Replaced by systemd-boot
+      - kernel # Replaced by kernel-ipe
     install:
-      - kernel-ipe
       - openssh-server
+      - syslog
+      - WALinuxAgent
+      # OS
       - device-mapper
-      # selinux
-      - policycoreutils-python-utils
-      - selinux-policy
-      - selinux-policy-modules
-      - selinux-policy-ci
-      - checkpolicy  
-      - secilc 
-      - setools-console
+      - kernel-ipe
+      # servicing
+      # - trident
+      # - trident-service
       - veritysetup
-      - vim
-      - WALinuxAgent
+      # OCI
+      - cni
       - containerd2
-      - ca-certificates
-      - wget
-      - syslog
-      - jq
-      - keyutils
       - cri-tools
-      - cni
-      - tardev-snapshotter
       - erofs-utils
-      - netplan
+      # - notation
+      - tardev-snapshotter
```

Comment:
We should discuss with this with Chris, whether there is a value in shipping the tardev-snapshotter at this point. I am still leaning towards yes.

Reply:
Given the meeting, I think we can also remove erofs-snapshotter.

Reply:
removed